game-core/src/main/java/games/strategy/triplea/ai/pro/util/ProTerritoryValueUtils.java

```diff
@@ -429,18 +431,41 @@ private static double findWaterValue(
     return capitalOrFactoryValue / 100 + nearbyLandValue / 10;
   }
 
+  // 1700 oldT=4906 newT=227 newT2=166 newT3=152
+  @SuppressWarnings("ReferenceEquality")
   private static Set<Territory> findNearbyEnemyCapitalsAndFactories(
-      final Territory t, final Map<Territory, Double> enemyCapitalsAndFactoriesMap) {
-
-    Set<Territory> nearbyEnemyCapitalsAndFactories = new HashSet<>();
-    for (int i = MIN_FACTORY_CHECK_DISTANCE; i <= MAX_FACTORY_CHECK_DISTANCE; i++) {
-      nearbyEnemyCapitalsAndFactories = t.getData().getMap().getNeighbors(t, i);
-      nearbyEnemyCapitalsAndFactories.retainAll(enemyCapitalsAndFactoriesMap.keySet());
-      if (!nearbyEnemyCapitalsAndFactories.isEmpty()) {
-        break;
+      final Territory t, final Set<Territory> enemyCapitalsAndFactories) {
```

Comment:
Is `t` equivalent to the `startTerritory`, the 'basis' for the search? I'm wondering if the algorithm perhaps would be a bit clearer if 't' and 't2' were spelled out a bit more.

Reply:
Done.